tests/test_covarites_assisted_embed.py

```diff
@@ -0,0 +1,1543 @@
+                0.0,
+                0.0,
+                0.0,
+                1.0,
+                0.0,
+                1.0,
+            ],
+            [
+                1.0,
+                1.0,
+                1.0,
+                1.0,
+                1.0,
+                1.0,
+                0.0,
+                1.0,
+                1.0,
+                0.0,
+                1.0,
+                0.0,
+                0.0,
+                1.0,
+                0.0,
+                1.0,
+                1.0,
+                0.0,
+                1.0,
+                0.0,
+            ],
+        ]
+        covarites = [
+            [0.0, 1.0],
+            [0.0, 1.0],
+            [0.0, 0.0],
+            [0.0, 1.0],
+            [1.0, 1.0],
+            [1.0, 1.0],
+            [1.0, 0.0],
+            [1.0, 1.0],
+            [0.0, 1.0],
+            [0.0, 1.0],
+            [1.0, 1.0],
+            [1.0, 1.0],
+            [1.0, 0.0],
+            [1.0, 0.0],
+            [1.0, 1.0],
+            [1.0, 1.0],
+            [0.0, 0.0],
+            [1.0, 0.0],
+            [1.0, 0.0],
+            [1.0, 0.0],
+        ]
+        labels = 10 * [0] + 10 * [1]
+
+        casc = CovariateAssistedSpectralEmbed(
+            n_components=2, assortative=False, CCA=False, check_lcc=False
+        )
+        casc_results = casc.fit(np.array(A), covarites)
+
+        results_ans = {
+            "cluster": ([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]),
+            "h": 0.015116721930096055,
+            "wcss": 0.0950368829932085,
+            "cascSvd": (
+                [
+                    [0.22082412, -0.14466871],
+                    [0.21389769, -0.1595033],
+                    [0.1886564, -0.18401221],
+                    [0.22020656, -0.23830119],
+                    [0.25428279, -0.22067716],
+                    [0.26337628, -0.14660874],
+                    [0.18420035, -0.27422263],
+                    [0.25436032, -0.16826633],
+                    [0.20181397, -0.30901171],
+                    [0.20098875, -0.33220764],
+                    [0.26779015, 0.22182241],
+                    [0.2408902, 0.21116545],
+                    [0.19211083, 0.29494827],
+                    [0.20147926, 0.23193894],
+                    [0.24204563, 0.2817552],
+                    [0.259063, 0.21670685],
+                    [0.17655257, 0.19310361],
+                    [0.21024992, 0.18047424],
+                    [0.21069076, 0.22192375],
+                    [0.23433936, 0.05816596],
+                ]
+            ),
+        }
+        ResultARI = ARI(casc_results["cluster"], results_ans["cluster"])
+        self.assertEqual(casc_results["h"], results_ans["h"])
+        self.assertEqual(casc_results["wcss"], results_ans["wcss"])
+        self.assertEqual(casc_results["cascSvd"], results_ans["cascSvd"])
+        self.assertEqual(ResultARI, 1)
+
+
+if __name__ == "__main__":
+    unittest.main()

```

Comment:
This is clearly not the correct way to write pytests. If you check travis, these tests are never ran.

Reply:
Got it .Now these lines are removed and a missing line  is updated.